brandingbrand/flagship
# feat: Add create-fs-app functionality

To test/run locally:

1. Navigate to `packages/create-fs-app`
2. Run `yarn link`
3. You should be able to run `create-fs-app` now in the directory of your choosing
